core-bundle/src/Resources/contao/dca/tl_content.php

```diff
@@ -1641,7 +1641,7 @@ public function getArticles(DataContainer $dc)
 		$arrPids = array();
 		$arrArticle = array();
 		$arrRoot = array();
-		$intPid = $dc->activeRecord->pid;
+		$intPid = $dc->activeRecord->pid ?? null;
```

Comment:
I still think the "fallback value" of a field named `intPid` should be `0` and not `null`

Reply:
But this would be a (potential) behavior change and we intentionally wanted to keep the code working the same as before for the PHP 8 warning patches.